vidyut-prakriya/examples/create_all_tinantas.rs

```diff
@@ -52,6 +53,7 @@ fn create_output_string(
     output_scheme: Scheme,
 ) -> String {
     items.sort();
+    items = items.into_iter().unique().collect();
```

Comment:
if you're using itertools for uniqueness, you can also call `items.dedup()` after sorting to remove duplicates.